include/rocksdb/file_system.h

```diff
@@ -653,7 +653,7 @@ class FileSystem : public Customizable {
   //
   // Default implementation is to return IOStatus::OK.
 
-  virtual IOStatus Poll(std::vector<IOHandle*>& /*io_handles*/,
+  virtual IOStatus Poll(std::vector<void*>& /*io_handles*/,
```

Comment:
Can we update the comment to indicate supporting Poll is mandatory? The Poll implementation should ensure that the callback gets called at IO completion, and return only after the callback has been called. This will simplify RocksDB implementation.